src/ui/account.js

```diff
@@ -180,12 +180,32 @@ cwc.ui.Account.prototype.setAuthentication = function(authenticated) {
   if (navigationInstance) {
     navigationInstance.enableOpenGoogleDriveFile(authenticated);
     navigationInstance.enableSaveGoogleDriveFile(authenticated);
+    navigationInstance.enableOpenGoogleClassroom(authenticated);
   }
 
   if (!authenticated) {
     this.accessToken = '';
   }
   this.authenticated = authenticated;
+
+  //Need to make classroom call after authentication is set.
+  if (navigationInstance) {
+    if(authenticated) {
```

Comment:
Any reason why this can't be moved into the former `if (navigatorInstance)` if the `this.authenticated = authenticated` is an issue, just move the block below.